F21/rjuliang/LowerCaser/assets/style.css

```diff
@@ -0,0 +1,47 @@
+/*
+This file is exclusively for CSS
+*/
+
```

Comment:
Can you please make it clear what these functions are for?

Reply:
Absolutely Bhargavi, CSS file (or aka stylesheet), tells the HTML file how to apply design properties to its elements. It is highly used for wed design. It does not affect any of the functionality in the back end of this project. (However, it usually does not affect the back end in any application).